CHANGELOG.md

```
@@ -1,5 +1,61 @@
 # Changelog
 
+## 1.16.0 (2026-01-07)
+
+## What's Changed
+
+### Refactoring (Week 1-2)
+
+- refactor: split chat/buffer.lua into focused modules by @shabaraba in https://github.com/shabaraba/vibing.nvim/pull/TBD
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)